jdaviz/configs/default/plugins/model_fitting/initializers.py

```diff
@@ -203,6 +222,51 @@ def _set_width_attribute(self, instance, name, fwhm):
         _setattr(instance, name, WIDTH, fwhm / 2.355)
 
 
+class _BlackBodyInitializer(object):
+    """
+    Initialization that is specific to the BlackBody model.
+
+    Notes
```

Comment:
I am not going to comment on the Sphinx stuff here because a hidden class will not make it to public API doc. FYI.